Kickstarter-iOS/Views/RewardCardContainerView.swift

```diff
@@ -125,21 +144,30 @@ public final class RewardCardContainerView: UIView {
     .flatMap { $0 })
   }
 
-  private func addPledgeButtonMarginConstraints(with layoutMarginsGuide: UILayoutGuide) {
+  private func addBottomViewsMarginConstraints(with layoutMarginsGuide: UILayoutGuide) {
     NSLayoutConstraint.deactivate(self.pledgeButtonMarginConstraints ?? [])
+    let minTouchSize = Styles.minTouchSize.height
 
     NSLayoutConstraint.activate([
       self.pledgeButton.leftAnchor.constraint(equalTo: layoutMarginsGuide.leftAnchor),
       self.pledgeButton.rightAnchor.constraint(equalTo: layoutMarginsGuide.rightAnchor),
       self.pledgeButton.bottomAnchor.constraint(lessThanOrEqualTo: layoutMarginsGuide.bottomAnchor),
-      self.pledgeButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Styles.minTouchSize.height)
+      self.pledgeButton.heightAnchor.constraint(greaterThanOrEqualToConstant: minTouchSize)
+    ])
+
+    NSLayoutConstraint.activate([
+      self.gradientView.leftAnchor.constraint(equalTo: layoutMarginsGuide.leftAnchor),
+      self.gradientView.rightAnchor.constraint(equalTo: layoutMarginsGuide.rightAnchor),
+      self.gradientView.topAnchor.constraint(equalTo: self.pledgeButton.topAnchor, constant: -minTouchSize),
```

Comment:
Looking at the gradient view with the UI inspector, I'd say it looks a teeeeeny bit big.

![image](https://user-images.githubusercontent.com/3156796/62320634-ab9b6180-b46e-11e9-8fc3-45412207a6fe.png)

From my understanding the purpose of the gradient is to fade text as it falls *below* the pledge button, basically to prevent de-emphasizing the pledge button. If that's the case then really we just want the gradient to wrap around the pledge button - my recommendation would be using something like `Styles.grid(2)` as a margin above and below the pledge button so that the gradient is only slightly larger than the pledge button in height.

Thoughts @dannyalright ?

Reply:
> From my understanding the purpose of the gradient is to fade text as it falls below the pledge button, basically to prevent de-emphasizing the pledge button.

That's correct, although fading one or two lines of the _above_ text helps with this goal. I.e. less-faded text above the green button takes away from the button's emphasis.

I think we should ship as is, but appreciate this deep dive!